packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -72,6 +72,13 @@ const devtoolsBlue = ColorSwatch<int>(600, {
   400: Color(0xFF13B9FD),
 });
 
+// Error color selected from
+// https://material.io/design/color/#color-theme-creation.
+const devtoolsError = Color(0xFFB00020);
+
+// Yellow 100 from https://material.io/design/color/#color-theme-creation.
+const devtoolsWarning = Color(0xFFFFF9C4);
```

Comment:
this color seems too bright for the dark theme.

Reply:
switched to colors from @raison00